CryptoLightNews/Model/CoinsModel.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  CoinsModel.swift
+//  CryptoLightNews
+//
+//  Created by Richard Price on 27/01/2023.
+//
+
+import Foundation
+
+struct CoinsModel: Codable {
+    var name: String?
+    var symbol: String?
+    var rank: Int?
+    var id: String?
+    var price_usd: String?
+    var market_cap_usd: String?
+    var percent_change_1h: String?
+    var percent_change_24h: String?
+    var percent_change_7d: String?
+}
+
+struct SearchResultsCoins: Codable {
```

Comment:
SearchResultsCoins implies you are doing an action with this model. We want to represent a group of Coins, so use a noun for that. Actions are for functions/methods.
```suggestion
struct Coins: Codable {
```